### DISCUSSION TITLE
Timer in seconds not showing "seconds" instead of "minute"

### DISCUSSION THREAD

**Discussion by AbdallahEssamGaber** [Hanziness/AnotherPomodoro#271] [Q&A - Development]
Surprisingly, I did need to use this seconds thing so..
If you set the timer to 5 seconds (the minimum) it still counts minutes, is it just me?

**Comment by Hanziness** [comment]
Whoops, I'm terribly sorry for not responding to this earlier, I'm unsure how I missed your question - which has since been probably already answered. 

The default "approximate" timer will not show seconds, it is by design to reduce updates on the screen and stop the user from constantly tracking the changes. If you need seconds to display, change the timer display in the *display* section of the settings to "Traditional" - it shows both minutes and seconds in a more familiar fashion.
